pom.xml

```diff
@@ -19,7 +19,14 @@
 
   <dependencies>
 
-    <dependency>
+      <dependency>
```

Comment:
Sample frameworks should not include misc additional dependencies that obscure their purpose -- to demonstrate using Sauce Labs.